### PR TITLE
Fix logic for resetTtl and adjust default

### PR DESCRIPTION
**Pull request by kibertoad**
See https://github.com/avoidwork/tiny-lru/pull/59#issuecomment-1466208804 for context

### CODE REVIEW

File: src/lru.js [hunk 1]
```diff
@@ -130,6 +126,9 @@ class LRU {
 					next.prev = prev;
 				}
 			}
+			if (resetTtl) {
```
Comment (kibertoad) [inline src/lru.js:129]: @avoidwork This was previously skipped for the last entry, so I moved it out of the `if not last entry` block, as it made no sense to keep it there.

Comment (avoidwork) [reply src/lru.js:129]: That's where it was with previous iteration 👍🏻

File: src/lru.js [hunk 2]
```diff
@@ -98,7 +98,7 @@ class LRU {
 		return Object.keys(this.items);
 	}
 
-	set (key, value, bypass = false, resetTtl = false) {
+	set (key, value, bypass = false, resetTtl = true) {
```
Comment (avoidwork) [inline src/lru.js:101]: this isn't what i meant earlier; it should come from an attribute like 'ttl'.

Comment (kibertoad) [reply src/lru.js:101]: you mean a numeric one?

**Comment by avoidwork** [comment]
See #66 ... now it's primed to change the default value of `resetTtl` on the factory & constructor for a major semver bump.

**Comment by kibertoad** [comment]
I'll create separate PR with tests from this branch, as they are still missing on master

**Comment by kibertoad** [comment]
@avoidwork I've checked your PR, and it has tests and everything, so nothing is needed anymore :)

**Pull request closed without merging**